src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -715,11 +714,9 @@ def get_action_option(cls, slack_request: SlackActionRequest) -> str | None:
         return action_option
 
     @classmethod
-    def get_action_list(
-        cls, slack_request: SlackActionRequest, use_block_kit: bool
-    ) -> list[MessageAction]:
+    def get_action_list(cls, slack_request: SlackActionRequest) -> list[MessageAction]:
         action_data = slack_request.data.get("actions")
-        if use_block_kit and action_data:
+        if action_data:
             # XXX(CEO): this is here for backwards compatibility - if a user performs an action with an "older"
```

Comment:
I think you can remove this comment now? Unless I'm reading it wrong

Reply:
idk you kept it in your PR, is action data both present in old and new slack notifications?